### DISCUSSION TITLE
How to make my particles as a background of a components?

### DISCUSSION THREAD

**Discussion by RichmondRamil** [matteobruni/tsparticles#2737] [Q&A] [answered]
const Skills = () => {
  return (
    <div className="mon__skills section__padding" id="skills">
      <Particles
        id="tsparticles"
        init={particlesInit}
        loaded={particlesLoaded}
        options={{
          fpsLimit: 120,
          interactivity: {
            events: {
              onClick: {
                enable: true,
                mode: "push",
              },
              onHover: {
                enable: true,
                mode: "repulse",
              },
              resize: true,
            },
            modes: {
              bubble: {
                distance: 800,
                duration: 2,
                opacity: 0.5,
                size: 1,
              },
              push: {
                quantity: 2,
              },
              repulse: {
                distance: 200,
                duration: 0.4,
              },
            },
          },
          particles: {
            color: {
              value: "#000",
            },
            links: {
              color: "#000",
              distance: 150,
              enable: true,
              opacity: 0.5,
              width: 1,
            },
            collisions: {
              enable: true,
            },
            move: {
              direction: "none",
              enable: true,
              outMode: "bounce",
              random: false,
              speed: 6,
              straight: false,
            },
            number: {
              density: {
                enable: true,
                value_area: 800,
              },
              value: 40,
            },
            opacity: {
              value: 0.5,
            },
            shape: {
              type: "circle",
            },
            size: {
              random: true,
              value: 5,
            },
          },
          detectRetina: true,
        }}
      />
      <div className="mon__skills-heading">
        <h1 className="gradient__text">
          This is the Front-End Development path that im taking. I choose
          ReactJS for JavaScript Library.
        </h1>
        <p>Find Development Path that suits to you</p>
        <div className="mon__skils-heading__beginner-container">
          <h1 className="gradient__text">Needs to learn First</h1>
          {beginnerSkills.map((item, index) => (
            <Feature
              title={item.title}
              text={item.text}
              key={item.title + index}
            />
          ))}
        </div>
      </div>
      <div className="mon__skills-container">
        <h1 className="gradient__text">Advance Learning</h1>
        {skillsData.map((item, index) => (
          <Feature
            title={item.title}
            text={item.text}
            key={item.text + index}
          />
        ))}
      </div>
    </div>
  );
};

export default Skills;

**Comment by matteobruni** [comment, marked as answer]
Duplicate of #2591
